Orange/canvas/utils/overlay.py

```diff
@@ -498,84 +544,105 @@ def button(self, standardButton):
 
 
 class NotificationOverlay(OverlayWidget):
-    # each canvas instance has its own overlay instance
-    overlayInstances = []
-
-    # list of queued notifications, overlay instances retain a clone of each notification
-    notifQueue = []
-
-    def __init__(self, parent=None, alignment=Qt.AlignRight | Qt.AlignBottom, **kwargs):
+    def __init__(self, parent, alignment=Qt.AlignRight | Qt.AlignBottom, **kwargs):
         """
         Registers a new canvas instance to simultaneously display notifications in.
-        The parent parameter should be the canvas' scheme widget.
+        The parent should be the canvas' scheme widget.
```

Comment:
Is there any reason that who should be the parent is specified. The notifications should be displayable over any widget that so desires.